androidshared/src/main/res/layout/app_bar_layout.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.appbar.AppBarLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/appBarLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:elevation="4dp">
+
+    <com.google.android.material.appbar.MaterialToolbar
+        android:id="@+id/toolbar"
+        style="@style/Widget.AndroidShared.Toolbar.Surface.HighEmphasisControls"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
Shouldn't this be `?attr/actionBarSize` like in https://material.io/components/app-bars-top/android#regular-top-app-bar?

Reply:
Yup. Good catch.